CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Bug fixes
+* Modified `skbio.stats.composition.ilr` and `skbio.stats.composition.ilr_inv` to suppress errors when inputing a 1D basis.  ([#1393](https://github.com/biocore/scikit-bio/issues/1393)).  Also fixing how the basis is handled in `skbio.stats.composition.ilr_inv`.
```

Comment:
Please update the second bug description to include what the previous behavior was (was it a silent bug that produced incorrect results, did it fail noisily, or something else?) and what the new behavior is.

In general, when a bug fix is submitted it is essential that the changelog entry includes these details as this is how we notify our users. This is analogous to QIIME 1's blog posts describing bugs (they include the same information for users).